### PR TITLE
Fix: Generate types when generating sdl

### LINKED ISSUE #3345: Schema types not generated after `rw g sdl`
### Whats the problem?
If you run `yarn rw generate sdl UserExample` - and try to use that graphql type in another sdl file, it will complain, if you don't have the dev server running.

This is because the schema we use for typechecking (and in the graphql plugin) - hasn't generated yet.

### Solution
At the end of sdl generation, run `yarn rw g types` similar to page/cell/scaffold generators.

### PR DESCRIPTION
**Pull request by callingmedic911**
Closes #3345.

Like @dac09 suggested, implemented exactly same of scaffold and other generators.

### CODE REVIEW

File: packages/cli/src/commands/generate/sdl/sdl.js [hunk 1]
```diff
@@ -225,6 +225,12 @@ export const handler = async ({ model, crud, force, tests, typescript }) => {
           return writeFilesTask(f, { overwriteExisting: force })
         },
       },
+      {
+        title: `Generating types ...`,
+        task: async () => {
```
Comment (Tobbe) [inline packages/cli/src/commands/generate/sdl/sdl.js:230]: Can't this be simplified to just be `task: generateTypes`? I realize all the other type genration tasks look the same, but if it works, see it as an opportunity to fix them all ;)

File: packages/cli/src/commands/generate/sdl/sdl.js [hunk 2]
```diff
@@ -225,6 +225,12 @@ export const handler = async ({ model, crud, force, tests, typescript }) => {
...
+          return generateTypes()
+        },
+      },
```
Comment (Tobbe) [inline packages/cli/src/commands/generate/sdl/sdl.js:233]: Can't this be simplified to just be task: generateTypes? I realize all the other type generation tasks look the same, but if it works, see it as an opportunity to fix them all ;)

```suggestion
        task: generateTypes
```

File: packages/cli/src/commands/generate/sdl/sdl.js [hunk 3]
```diff
@@ -225,6 +225,12 @@ export const handler = async ({ model, crud, force, tests, typescript }) => {
           return writeFilesTask(f, { overwriteExisting: force })
         },
       },
+      {
+        title: `Generating types ...`,
+        task: async () => {
+          return generateTypes()
+        },
```
Comment (Tobbe) [inline packages/cli/src/commands/generate/sdl/sdl.js:232]: Can't this be simplified to just be task: generateTypes? I realize all the other type generation tasks look the same, but if it works, see it as an opportunity to fix them all ;)

```suggestion
        task: generateTypes
```

Comment (callingmedic911) [reply packages/cli/src/commands/generate/sdl/sdl.js:232]: Totally!

**Comment by dac09** [comment]
I know it seems like a small one - but does Listr still wait for generation to finish?

I'm wondering because generateTypes is an async function, I suppose Listr is smart enough to wait when it receives a promise?

**Comment by callingmedic911** [comment]
@dac09 Yes, verified this one. :) https://github.com/SamVerschueren/listr#promises

**Review by dac09 — APPROVED** [review]

**Pull request merged**